static/app/views/starfish/components/slideOverPanel.tsx

```diff
@@ -12,21 +11,22 @@ const PANEL_WIDTH = '50vw';
 type SlideOverPanelProps = {
   children: React.ReactNode;
   collapsed: boolean;
+  onOpen?: () => void;
 };
 
 export default forwardRef(SlideOverPanel);
 
 function SlideOverPanel(
-  {collapsed, children}: SlideOverPanelProps,
+  {collapsed, children, onOpen}: SlideOverPanelProps,
   ref: ForwardedRef<HTMLDivElement>
 ) {
   const {query} = useLocation();
   const organization = useOrganization();
   useEffect(() => {
-    if (!collapsed) {
-      trackAnalytics('starfish.panel.open', {organization});
+    if (!collapsed && onOpen) {
+      onOpen();
     }
-  }, [query, collapsed, organization]);
+  }, [query, collapsed, organization, onOpen]);
```

Comment:
Hey @edwardgou-sentry, do you know why `query` is included here? It's not being used in the `useEffect` and it's actually causing this hook to trigger twice in prod right now. The `collapsed` variable triggers a run, but there's also one that triggers for the `query` changing

Reply:
@narsaynorath I believe this was to trigger analytics if the query changes, which causes content in the panel to change.

In the case of starfish, you click an endpoint in the span summary page, that opens the slideout panel and updates teh url query.
You click another endpoint in the span summary, that would keep the existing slide out panel and update the content.

Reply:
Yeah we changed that recently, so there's no scenario where you can change the content on the sidebar while it's already opened.